sandbox/docker-compose.yml

```diff
@@ -47,8 +47,6 @@ services:
     depends_on:
       - oef-node
       - visdom
-    volumes:
```

Comment:
are you sure we do not need it anymore?

Reply:
the `volumes` section was duplicate, see below